packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -542,7 +566,7 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
       const barHeight: number = Math.max(yBarScale(point.y), 0);
       let adjustedBarHeight = 0;
       if (barHeight <= 0) {
-        return <React.Fragment key={point.x}> </React.Fragment>;
+        return <React.Fragment key={point.x instanceof Date ? point.x.getTime() : point.x}> </React.Fragment>;
```

Comment:
>instanceof [](http://example.com/codeflow?start=44&length=10)

This will always be numeric as it is part of the numeric function

Reply:
This is fixed